DeviceAdapters/OpenFlexure/OpenFlexure.cpp

```diff
@@ -1,13 +1,27 @@
-﻿//////////////////////////////////////////////////////////////////////////////
+///////////////////////////////////////////////////////////////////////////////
 // FILE:          OpenFlexure.cpp
 // PROJECT:       Micro-Manager
 // SUBSYSTEM:     DeviceAdapters
 //-----------------------------------------------------------------------------
-// DESCRIPTION:   Adapter for the OpenFlexure Microscope. This adapter is used on the v5 Sangaboard.
+// DESCRIPTION:   Adapter for the OpenFlexure Microscope. This adapter is used on the v5 Sangaboard. 
+//                
+// AUTHOR:        Samdrea Hsu, samdreahsu@gmail.com, 06/22/2024
+//                
+//                Karl Hoover (stuff such as programmable CCD size  & the various image processors)
```

Comment:
@samdrea did you mean to include these other authors?

Reply:
@henrypinkard No, I'm not sure how that got in there... Could you edit them off or should I open a new pr?

Reply:
If you make new commits and push again they should be added to the PR